test/xchain-oracle-ticker.test.ts

```diff
@@ -186,32 +97,35 @@ describe('xchainOracleTicker', function () {
             expect(callData).to.deep.equal([
                 {
                     to: userArgs.forwarder,
-                    data: forwarderArbitrum.interface.encodeFunctionData('refresh', [
-                        userArgs.gasLimit,
-                        userArgs.maxFeePerGas,
-                        userArgs.baseFee,
-                    ]),
+                    data: forwarder.interface.encodeFunctionData('refresh', refreshArgs),
                 },
             ])
 
-            const dsrBefore = (await forwarderArbitrum.getLastSeenPotData()).dsr
+            const dsrBefore = (await forwarder.getLastSeenPotData()).dsr
             expect(dsrBefore).to.not.equal(newDsr)
 
-            await keeper.sendTransaction({
-                to: callData[0].to,
-                data: callData[0].data,
-                value: BigInt('1000000000000000000'),
-            })
+            await insistOnExecution(() =>
+                keeper.sendTransaction({
+                    to: callData[0].to,
+                    data: callData[0].data,
+                }),
+            )
 
-            const dsrAfter = (await forwarderArbitrum.getLastSeenPotData()).dsr
+            const dsrAfter = (await forwarder.getLastSeenPotData()).dsr
             expect(dsrAfter).to.equal(newDsr)
         })
 
         it('refresh is needed (stale rho)', async () => {
+            const maxDelta = (
```

Comment:
Need to have failure case before this to demonstrate boundary

Reply:
Fixed. Unfortunately I need to leave 1 second of margin, because the time moves and if I test super tightly, the test is not stable.